projects/backend/controllers/editions.ts

```diff
@@ -1,21 +1,23 @@
 import { Request, Response } from 'express'
-import { SpecialEdition, EditionsList } from '../../Apps/common/src'
-import { defaultRegionalEditions } from '../../Apps/common/src/editions-defaults'
-import { s3Put } from '../s3'
+import { EditionsList } from '../../Apps/common/src'
+import { s3Put, s3fetch } from '../s3'
+import { isPreview } from '../preview'
+import { hasFailed } from '../utils/try'
 
-const getSpecialEditions = (): Array<SpecialEdition> => {
-    return []
-}
-
-const editionsList: EditionsList = {
-    regionalEditions: defaultRegionalEditions,
-    specialEditions: getSpecialEditions(),
-    trainingEditions: [],
-}
+export const editionsControllerGet = async (req: Request, res: Response) => {
+    const editionsList = await s3fetch({
+        key: 'editionsList',
```

Comment:
should this be `editions`?

Reply:
as discussed on video call - this file is in the cmsFronts account rather than editions and has a different name there.